_people-use-web/user-stories.es.md

```diff
@@ -0,0 +1,403 @@
+
+La hija de Yun le regaló un ratón especial que le resulta más fácil de usar con su temblor de manos. También utiliza un navegador que guarda imágenes en miniatura de los sitios web que visita con frecuencia, de esta forma es más fácil para el identificar los sitios a los que más acude.
+
+Vea los siguientes recursos donde encontrará más información para implementar técnicas que eliminan las barreras que encuentra Yun:
+
+{% include_cached excol.html type="start" id="yun-sections" %}
+
+### Secciones relacionadas con Yun
+
+{% include_cached excol.html type="middle" %}
+
+**Habilidades y barreras diversas:**
+
+-   [Baja visión (discapacidades visuales)](/people-use-web/abilities-barriers/#visual)
+-   [Temblor de manos (discapacidades físicas)](/people-use-web/abilities-barriers/#physical)
+-   [Pérdida de memoria a corto plazo leve (discapacidades cognitivas disabilities)](/people-use-web/abilities-barriers/#cognitive)
+
+**Herramientas y técnicas:**
+
+-   [Magnificación de la pantalla (presentación)](/people-use-web/tools-techniques/#presentation)
+-   [Teclado y ratón alternativos (entrada)](/people-use-web/tools-techniques/#input)
+-   [Filtros de teclado y ratón (entrada)](/people-use-web/tools-techniques/#input)
+-   [Personalización del ratón (entrada)](/people-use-web/tools-techniques/#input)
+-   [Marcadores e historia (interacción)](/people-use-web/tools-techniques/#interaction)
+-   [Consistencia y predicción (interacción)](/people-use-web/tools-techniques/#interaction)
+-   [Títulos, encabezados y etiquetas descriptivos (interacción)](/people-use-web/tools-techniques/#interaction)
+-   [Mensajes de error y de éxito útiles (interacción)](/people-use-web/tools-techniques/#interaction)
+
+**Principios de accesibilidad:**
+
+-   [Alternativas textuales para contenido no textual (Perceptible)](/fundamentals/accessibility-principles/#alternatives)
+-   [El contenido se puede presentar de diferentes formas (Perceptible)](/fundamentals/accessibility-principles/#adaptable)
+-   [El contenido es más fácil de ver y de oír (Perceptible)](/fundamentals/accessibility-principles/#distinguishable)
+-   [Los usuarios tienen suficiente tiempo para leer y usar el contenido (Operable)](/fundamentals/accessibility-principles/#time)
+-   [Users can easily navigate, find content, and determine where they are (Operable)](/fundamentals/accessibility-principles/#navigable)
```

Comment:
In English.
Los usuarios pueden navegar, encontrar contenido y determinar donde se encuentran de forma sencilla (Operable)